app/scripts/modules/amazon/src/loadBalancer/loadBalancer.transformer.js

```diff
@@ -29,31 +29,35 @@ module.exports = angular.module('spinnaker.aws.loadBalancer.transformer', [
       };
     }
 
-    function transformInstance(instance, loadBalancer) {
+    function transformInstance(instance, provider, account, region) {
       instance.health = instance.health || {};
-      instance.provider = loadBalancer.type;
-      instance.account = loadBalancer.account;
-      instance.region = loadBalancer.region;
-      instance.health.type = 'LoadBalancer';
+      if (instance.health.state === 'healthy') {
+        // Target groups use 'healthy' instead of 'InService' and a lot of deck expects InService
+        // to surface health in the UI, just set it as InService since we don't really care the
+        // specific state name... yet
+        instance.health.state = 'InService';
+      }
+      instance.provider = provider;
+      instance.account = account;
+      instance.region = region;
       instance.healthState = instance.health.state ? instance.health.state === 'InService' ? 'Up' : 'Down' : 'OutOfService';
       instance.health = [instance.health];
-      instance.loadBalancers = [loadBalancer.name];
     }
 
-    function addVpcNameToLoadBalancer(loadBalancer) {
+    function addVpcNameToContainer(container) {
       return function(vpcs) {
-        var matches = vpcs.filter(function(test) {
-          return test.id === loadBalancer.vpcId;
+        var matches = vpcs.filter((test) => {
```

Comment:
can use `find` instead of `filter`, then do `match ? match.name : '';` below.

(bad pattern; I thought I'd cleaned up most of it but obviously missed this one)